tarsplitter.go

```diff
@@ -11,13 +11,16 @@ import (
 	"path/filepath"
 	"strings"
 	"sync"
+	"github.com/c2h5oh/datasize"
```

Comment:
Thanks for the contribution!


How would you feel about making this megabytes, and ditch the dependency?